src/ui/window/ShortcutManager.qml

```diff
@@ -35,6 +35,21 @@ Item {
     property SearchOverlay searchOverlay
     property var tabsModel: shortcutManager.window.tabsModel
 
+    Connections {
```

Comment:
One last thing: to follow [QML conventions](http://doc.qt.io/qt-4.8/qml-coding-conventions.html), can you please move the `Connections` element below the `setActiveTabByIndex` function?